# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/mine/no-disk.patch
# Copyright (C) 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- mine-0.23/gasgui.c.vanilla	2022-11-15 14:09:55.412607302 +0100
+++ mine-0.23/gasgui.c	2022-11-15 14:10:24.029604324 +0100
@@ -204,8 +204,7 @@
 		printf("%s\n", command);
 		fflush(stdout);
 	} else {
-		printf("Disk #%d: Installing %s-%s ...\n",
-		       p->disk_number, p->name, p->version);
+		printf("Installing %s-%s ...\n", p->name, p->version);
 		fflush(stdout);
 		rc = system(command);
 
